extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/logs/OpenTelemetryLogConfig.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.opentelemetry.runtime.logs;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+
+@ConfigMapping(prefix = "quarkus.log.handler.open-telemetry")
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public interface OpenTelemetryLogConfig {
+    /**
+     * Determine whether to enable the OpenTelemetry logging handler
+     */
+    boolean enabled();
```

Comment:
not sure this should be configurable... if OTel logging is ON, this should be implicit...

Reply:
OTel login is automatically on (as for metrics and traces), but someone may want to add opentelemetry but only enable login in production.

This is a runtime config and it's the same for all the other logging handler so better to be consistent.

Reply:
This appender is OTel specific and we can argue that setting the exporter to none at runtime, has the same effect. 
An alternative is to print a INFO console log entry saying that no logs will be sent to OTel Logging because the appender is OFF... 
I already see some users having one config switched on, the other off and then complaining that OTel Logging doesn't work...

Reply:
> setting the exporter to none at runtime, has the same effect.

`quarkus.otel.logs.enabled` is a build time config as for `quarkus.otel.logs.exporter`
Which runtime config do your talk about ?

Reply:
`quarkus.otel.logs.exporter` is a runtime config